packages/bee-q/src/components/breadcrumb/bq-breadcrumb.tsx

```diff
@@ -0,0 +1,105 @@
+import { h, Component, Host, Element, Listen, Event, EventEmitter, Prop } from '@stencil/core';
+
+import { isHTMLElement } from '../../shared/utils';
+
+@Component({
+  tag: 'bq-breadcrumb',
+  styleUrl: './scss/bq-breadcrumb.scss',
+  shadow: true,
+})
+export class BqBreadcrumb {
+  // Own Properties
+  // ====================
+
+  // Reference to host HTML element
+  // ===================================
+
+  @Element() el!: HTMLBqBreadcrumbElement;
+
+  // State() variables
+  // Inlined decorator, alphabetical order
+  // =======================================
+
+  // Public Property API
+  // ========================
+
+  /** The icon name used as separator. Default is `/`. */
+  @Prop() separatorIcon!: string;
+
+  // Prop lifecycle events
+  // =======================
+
+  // Events section
+  // Requires JSDocs for public API documentation
+  // ==============================================
+
+  /** Handler to be called when `bq-breadcrumb-item` item loses focus. */
+  @Event() bqBreadcrumbBlur: EventEmitter<HTMLBqBreadcrumbItemElement>;
+
+  /** Handler to be called when `bq-breadcrumb-item` item gets focus. */
+  @Event() bqBreadcrumbFocus: EventEmitter<HTMLBqBreadcrumbItemElement>;
+
+  /** Handler to be called when `bq-breadcrumb-item` is selected (on click/enter press). */
+  @Event() bqBreadcrumbClick: EventEmitter<HTMLBqBreadcrumbItemElement>;
+
+  // Component lifecycle events
+  // Ordered by their natural call order
+  // =====================================
+
+  componentDidLoad() {
+    this.setIconAsSeparator();
```

Comment:
I would add `@Watch('separatorIcon')`. If they change dynamically the separator we would want to update it automatically.